### DISCUSSION TITLE
using own API login data

### DISCUSSION THREAD

**Discussion by thommierother** [flobz/psa_car_controller#144] [Q&A] [answered]
Aside from using the imported credentials from the mypeugeot app:
1. For development, is it possible to use your own user account (including client id/secret) on the psa API?
2. what APIs are needed to be registered for the app?
- connected-car-api (1.0.0) (default) 
- others?

And I can not see apiv4 there ... ??

Tx, THommie

**Comment by flobz** [comment, marked as answer]
Hello,
PSA doesn't propose an API for developer yet.That's why we use the one used by Android app. Clientid and client secret are taken from the android app also.
